docs/knowledge_base/demos/churn_classification.md

```diff
@@ -0,0 +1,48 @@
+In the fast-paced world of business, retaining customers is a top priority. The Churn 
+Classification demo, powered by Taipy, is here to revolutionize customer retention. In this demo,
```

Comment:
Revolutionize customer retention is a bit ostentatious to me. 
We help analyzing churn, we don't do anything about retention directly.

```suggestion
Classification demo, powered by Taipy, is here to revolutionize 
[customer churn](https://en.wikipedia.org/wiki/Customer_attrition) analysis . In this demo,
```